svea_vision/launch/zed_main.launch

```diff
@@ -17,9 +17,10 @@
     <arg name="depth_image"     value="/zed/zed_node/depth/depth_registered"/>
     <arg name="image"           value="/zed/zed_node/rgb/image_rect_color"/>
 
+    <arg name="map_path" default="$(find svea_core)/maps/sml.yaml"/>
```

Comment:
Also, what do you think about adding arguments to specify if state estimation should start or not? This could be extended to other nodes as well. Basically, more granular control over which nodes that we want to start?

Reply:
Yes, I do like this. When possible, can you create a PR on main repo with suggested changes to our launch files?

Let's do default empty string for map_path.